templates/definition/meter/kostal-plenticore-ng.yaml

```diff
@@ -1,22 +1,22 @@
-template: kostal-plenticore-hw0200
+template: kostal-plenticore-ng
 products:
   - brand: Kostal
     description:
-      generic: Plenticore Hybrid (Gen2)
+      generic: Plenticore Hybrid (nextgen)
 capabilities: ["battery-control"]
 linked:
   - template: kostal-ksem-inverter
     usage: grid
   - template: kostal-ksem
     usage: grid
     excludetemplate: kostal-ksem-inverter
-covers: ["kostal-plenticore-gen2"]
+covers: ["kostal-plenticore-hw0200", "kostal-plenticore-gen2"]
 requirements:
   description:
     de: |
-      Nur ein System kann und darf auf den Wechselrichter zugreifen! Für die aktive Batteriesteuerung muss die externe Batteriesteuerung über Modbus mit dem Handwerkerzugang aktiviert sein. Das Netzladen steht mit dieser Vorlage zur Verfügung, ist aktuell jedoch inkompatibel mit Wechselrichtern der HW-Version 0100).
+      Nur ein steuerndes System kann und darf auf den Wechselrichter per Modbus zugreifen! Für die aktive Batteriesteuerung muss die Funktion externe Batteriesteuerung über Modbus mit dem Handwerkerzugang aktiviert sein. Das Netzladen der Batterie steht mit dieser Vorlage zur Verfügung, ist aktuell jedoch inkompatibel mit älteren Wechselrichtern (z.B. häufig Hardwareversion 0100). Es ist auf aktuelle Firmwarestände der Geräte zu achten. Die Steuerung des Wechselrichters erfolgt über das Modbus-Register 1028 (relative Ladeleistung).
     en: |
-      Only a single system may access the inverter! For active battery control, the external battery control via Modbus must be activated using installer access. The function for grid charging the battery is available using this template, but is currently incompatible with inverters of HW version 0100.
+      Only a single controlling system may access the inverter via modbus! For active battery control, the feature external battery control via modbus must be activated using installer access. The function for grid charging the battery is available using this template, but is currently incompatible with some old inverters (for example mostly hardware version 0100). Make sure you have the latest firmware versions for all devices. Modbus register 1028 (relative charge rate) is used to control the device.
```

Comment:
Is there a particular reason for this? What does it mean specifically? You must always have only one controlling instance.

Reply:
Bitte raus. Das gilt für ganz evcc.